server/compute/run_diffexp.py

```diff
@@ -0,0 +1,88 @@
+import sys
```

Comment:
seems like this belongs in the test directory, or somewhere else that will prevent it from being shipped as part of the package.